### DISCUSSION TITLE
Change color of cover card

### DISCUSSION THREAD

**Discussion by MBFS12B** [Clooos/Bubble-Card#663] [Questions about config, custom styles and templates]
Hi, changing colors of switches, sliders and so on works fine for me. I'm just to stupid to change the color of the cover card. Non of the examples in https://github.com/Clooos/Bubble-Card?tab=readme-ov-file#styling seem to work for that. Can anyone help me out? Markus

**Comment by MrBearPresident** [comment]
Did you solve this? 
Can this be closed?
